Packs/NucleonCyber/Integrations/NucleonCyberFeed/command_examples

```diff
@@ -0,0 +1 @@
+!helloworld-get-indicators limit=100
```

Comment:
Please add command examples of your integration, and run
demisto-sdk generate-docs -i <Path to NucleonCyberFeed.yml> 
and regenerate README with all the command examples.